injectors/index.js

```diff
@@ -73,13 +98,29 @@ List of valid platforms:\n${AnsiEscapes.GREEN}${VALID_PLATFORMS.map(x => `${x}`)
     }
   } else if (process.argv[2] === 'uninject') {
     try {
-		result = await main.uninject(platformModule, platform);
-	} catch (e) {
-		// this runs if path generator crashes (app folder doesnt exist)
-		console.log(`${AnsiEscapes.RED}Platform you specified isn't installed on this device!${AnsiEscapes.RESET}\n\nList of valid platforms:\n${AnsiEscapes.GREEN}${VALID_PLATFORMS.map(x => `${x}`).join('\n')}${AnsiEscapes.RESET}`);
-		process.exit(process.argv.includes('--no-exit-codes') ? 0 : 1);
-	}
-	if (result) {
+      if (!process.argv.find(x => VALID_PLATFORMS.includes(x.toLowerCase()))) {
```

Comment:
Let's make a shared function to determine the platform so we don't have to rewrite the same code.